components/overview/AmountInput.js

```diff
@@ -12,12 +11,30 @@ const AmountInputDefault = ({
 	availableBalance,
 	onChange,
 	networkInfo,
+	isExistingUser,
```

Comment:
@Lord-of-Codes isExistingUser will always be undefined here, since it is not being sent as a prop

Reply:
@sahilnanda1995 It is being sent from InvestMoreModal, and its only going to be used when the transactionType is "bond". So, no problem would be caused.